src/test/java/LoginTests.java

```diff
@@ -1,115 +1,62 @@
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
-        emailInput.click();
-        emailInput.sendKeys("demo@class.com");
-        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitLogin.click();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        Assert.assertTrue(submitLogin.isDisplayed());
-        WebElement registationLink = driver.findElement(By.id("hel"));
-        Assert.assertTrue(registationLink.isDisplayed(), "==== Registation link displayed ====");
-        driver.quit();
+    public static void LoginWrongPasswordTest() {
+        LoginPage loginPage = new LoginPage(getThreadDriver());
+
+        loginPage.provideEmail("ksenia.strigkova@gmail.com")
+                .providePassword("te$t123")
+                .clickSubmitBtn();
+
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
     }
 
     @Test
-    public static void LoginWrongEmailTest () {
-
-        WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://bbb.testpro.io/";
-        driver.get(url);
-        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
-        emailInput.click();
-        emailInput.sendKeys("dem@class.com");
-        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
-        passwordInput.click();
-        passwordInput.sendKeys("te$t$tudent");
-        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
-        submitLogin.click();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        Assert.assertTrue(submitLogin.isDisplayed());
-        WebElement registationLink = driver.findElement(By.id("hel"));
-        String link = registationLink.getText();
-        System.out.println("==== This is our link text ==== " + link);
-        driver.quit();
+    public static void LoginEmptyPasswordTest() {
+        LoginPage loginPage = new LoginPage(getThreadDriver());
+
+        loginPage.provideEmail("ksenia.strigkova@gmail.com")
+                .providePassword("")
+                .clickSubmitBtn();
+
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
     }
 
+    @Test
+    public static void LoginWrongEmailTest() {
+        LoginPage loginPage = new LoginPage(getThreadDriver());
 
+        loginPage.provideEmail("ksenia.strigkova@gmail.c")
+                .providePassword("te$t$tudent")
+                .clickSubmitBtn();
+
+        Assert.assertTrue(loginPage.getRegistrationLink().isDisplayed());
+    }
 }
-//        Email("demo@class.com");
-//        Password("te$t$tudent");
\ No newline at end of file
+---
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221006404-3c997b5d-b447-4b4a-8ebf-e48bcff36785.png)